terraform/modules/iam_user/read_only_admin/user.tf

```diff
@@ -0,0 +1,169 @@
+
+module "read_only_admin" {
+    source = ".."
+
+    username = "${var.username}"
+
+    /* TODO: Make the bucket names configurable */
```

Comment:
Looks like we don't refer to bucket names or `subgrp` here. Are these TODOs relevant?